standards/core/mix.md

```diff
@@ -0,0 +1,119 @@
+---
+title: WAKU-MIX
+name: Waku Mix
+editor: Prem Chaitanya Prathi <premprathi@proton.me>
+contributors:
+ - Akshaya Mani <akshaya@status.im>
+ - Hanno Cornelius <hanno@status.im>
+---
+
+# Abstract
+This document explains how [libp2p mix](https://rfc.vac.dev/vac/raw/mix/) protocol will be integrated into waku. This would provide higher anonymity for users publishing or querying for messages to/from the Waku network. 
+
+This document covers integration of mix with [lightpush](https://rfc.vac.dev/waku/standards/core/19/lightpush) and [store](https://rfc.vac.dev/waku/standards/core/13/store) protocols. It also covers the aspect of relay nodes acting as mix nodes.
+Both of these are request response based protocols that follow a service-user and service-provider model.
+
+Integration of gossipsub with mix is not covered in this document.
+
+## Background / Rationale / Motivation
+
+Waku protocols have weak sender/originator anonymity as explained in [Waku Privacy and Anonymity Analysis](https://vac.dev/rlog/wakuv2-relay-anon/).
+As of today, it is easy to analyze the network traffic and determine originator of messages published into the network or determine the topic interests by analyzing store queries requested by a user. Similar analysis is possible for topic subscriptions via [Filter](https://rfc.vac.dev/waku/standards/core/12/filter) protocol as well.
+
+By using the libp2p mix protocol, we can improve the anonymity for publishers and store query users in the waku network.
+Anonymity of Filter users will not be addressed by this and will have to be addressed separately.
+
+
+# Specification
+
+Mix protocol integration with waku would end up creating a separate mix network of all the waku nodes that also support the mix protocol. All mix packets would be routed via this mix network before reaching their intended destination.
+Nodes shall advertise that they support mix protocol via their [ENR](#ENR updates).
+Nodes that would like to use mix service shall discover enough mix nodes so that there is always a healthy pool of mix nodes available for selection. The pool size of mix nodes should be in multiples of 100 for the mixing to be effective.
+Nodes that want higher anonymity while publishing a message or performing a store query would use the mix protocol to route their messages to their destination.
+A response/reply to a protocol request sent using Mix can be implemented using `Single Use reply blocks` as the service node will not be aware of the originator of the request.
+In order to have better reliability, any node that wish to send message using mix shall use 2 redundant paths.
```

Comment:
I'd consider phrasing these with the following sentence template in mind

<Explicit agent> MUST/SHOULD/MAY <perform explicit action> (optional: to explicit object/under explicit condition/etc.)

In other words, use the RFC keywords as guideline to think about declarative statements that an implementer of Waku protocols can follow as a "recipe" to decide how and where to include mix. Example suggestions:

Nodes with `mix` protocol mounted SHOULD advertise that they support mix via their chosen discovery method.
They MAY do so by updating....their ENR according to section...and using one of the ENR-based discovery methods, e.g....
Nodes that would like to use `mix`, SHOULD discover enough mix nodes...
The pool size SHOULD be larger than...